BlueEastCode/bluerain-plugin-native-base
# fix(Upgraded bluerain-ui-interfaces and bluerain-os version, Added Ch…

…eckbox component with README.md

## What I did
Added Checkbox component
Added stories
Added README.md
Added new versions of bluerain-ui-interfaces, bluerain-platform-xp and bluerain-os in package.json
## How to test
Cannot test at the moment
Is this testable with jest or storyshots?
No
Are any of the existing tests are failing ?
No
Does this need an update to the documentation?
No
Does it have breaking changes ?
No
If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
